test/helpers/PageTitleSpec.scala

```diff
@@ -0,0 +1,40 @@
+/*
+ * Copyright 2023 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package repositories
+
+import utils.SpecBase
+import views.helpers.PageTitle
+import play.api.i18n.Messages
+import play.api.test.Helpers
+
+class PageTitleSpec extends SpecBase {
+  "PageTitle" should {
+    "pageTile should return correct string for present title" in new Setup {
```

Comment:
This could be updated as below cause should is already there on line 25. Same for line 31

"return correct string for present title"